src/data/supplies.json

```diff
@@ -28,50 +151,626 @@
     "isIngredient": true
   },
   {
-    "id": "mleko",
-    "name": "Mleko",
-    "unit": "mililitry",
-    "category": "nabiał",
+    "id": "pesto_czerwone",
+    "name": "Pesto czerwone",
+    "unit": "gramy",
+    "category": "inne",
+    "isVegetarian": true,
+    "isVegan": true,
+    "isIngredient": true
+  },
+  {
+    "id": "platki_sniadaniowe",
+    "name": "Płatki śniadaniowe",
+    "unit": "gramy",
+    "category": "inne",
+    "isVegetarian": true,
+    "isVegan": true,
+    "isIngredient": true
+  },
+  {
+    "id": "proszek_do_pieczenia",
+    "name": "Proszek do pieczenia",
+    "unit": "gramy",
+    "category": "inne",
+    "isVegetarian": true,
+    "isVegan": true,
+    "isIngredient": true
+  },
+  {
+    "id": "serek_kanapkowy",
+    "name": "Serek kanapkowy (Hochland itd.)",
+    "unit": "gramy",
+    "category": "inne",
+    "isVegetarian": true,
+    "isVegan": false,
+    "isIngredient": true
+  },
+  {
+    "id": "serek_topiony",
+    "name": "Serek topiony",
+    "unit": "sztuki",
+    "category": "inne",
+    "isIngredient": true,
+    "isVegetarian": true,
+    "isVegan": true
+  },
+  {
+    "id": "tortellini",
+    "name": "Tortellini",
+    "unit": "gramy",
+    "category": "inne",
+    "isIngredient": true,
+    "isVegetarian": false,
+    "isVegan": false
+  },
+  {
+    "id": "wege_pasta",
+    "name": "Wege pasta do kanapek",
+    "unit": "gramy",
+    "category": "inne",
+    "isVegetarian": true,
+    "isVegan": true,
+    "isIngredient": true
+  },
+  {
+    "id": "worki_do_lodu",
+    "name": "Woreczki do lodu",
+    "description": "Do zrobienia lodu do różnych rzeczy",
+    "unit": "sztuki",
+    "category": "inne",
+    "isIngredient": false
+  },
+  {
+    "id": "zapalniczka",
+    "name": "Zapalniczka",
+    "description": "Zapalniczka do kuchenki, iskrownik rzadko działa :(",
+    "unit": "sztuki",
+    "category": "inne",
+    "isIngredient": false
+  },
+  {
+    "id": "boczek",
+    "name": "Boczek wędzony",
+    "unit": "gramy",
+    "category": "mięso",
+    "isVegetarian": false,
+    "isVegan": false,
+    "isIngredient": true
+  },
+  {
+    "id": "kielbasa",
+    "name": "Kiełbasa",
+    "unit": "gramy",
+    "category": "mięso",
+    "isVegetarian": false,
+    "isVegan": false,
+    "isIngredient": true
+  },
+  {
+    "id": "kurczak_filety",
+    "name": "Filet z kurczaka",
+    "unit": "gramy",
+    "category": "mięso",
+    "isIngredient": true,
+    "isVegetarian": false,
+    "isVegan": false
+  },
+  {
+    "id": "parowka",
+    "name": "Parówka",
+    "unit": "gramy",
+    "category": "mięso",
+    "isVegetarian": false,
+    "isVegan": false,
+    "isIngredient": true
+  },
+  {
+    "id": "mieso_mielone_wolowe",
+    "name": "Mięso mielone wołowe",
+    "unit": "gramy",
+    "category": "mięso",
+    "isVegetarian": false,
+    "isVegan": false,
+    "isIngredient": true
+  },
+  {
+    "id": "wędlina_prozniowo",
+    "name": "Wędlina pakowana próżniowo",
```

Comment:
Jest tu sens oznaczać to jako pakowana próżniowo? Może po prostu wędlina do kanapek?